firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/Chat.kt

```diff
@@ -70,9 +78,17 @@ public class Chat(
   }
 
   /**
-   * Generates a response from the backend with the provided text prompt.
+   * Sends a message using the existing history of this chat as context and the provided text
+   * prompt.
+   *
+   * If successful, the message and response will be added to the history. If unsuccessful, history
+   * will remain unchanged.
    *
-   * @param prompt The text to be converted into a single piece of [Content] to send to the model.
+   * This function should be called within a suspend context to properly manage concurrency.
+   *
```

Comment:
Kind of a NOP, since it's a `suspend` function; it can only be ran from a suspend context.

```suggestion
```